src/content/en/updates/2017/06/play-request-was-interrupted.md

```diff
@@ -47,7 +47,8 @@ The code above results in this error message in Chrome DevTools:
 > call to pause()._
 
 As the video is not loaded due to `preload="none"`, video playback doesn't
-necessarily start immediately after `video.play()` is executed.
+necessarily start immediately after `video.play()` is executed. Note that the
+`playing` event is fired on the video when it begins to play.
```

Comment:
That's correct. You might want to say that the event is fired when the promise is resolved. Not sure if that helps. The idea was that it might be interesting for developers to understand that the promise resolving and the playing event are the same thing.